addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -940,10 +941,23 @@ const Wysiwyg = Widget.extend({
      * @param {jQuery} $editable
      * @returns {Promise}
      */
-    saveModifiedImages: function ($editable = this.$editable) {
+    savePendingImages($editable = this.$editable) {
         const defs = _.map($editable, async editableEl => {
             const {oeModel: resModel, oeId: resId} = editableEl.dataset;
-            const proms = [...editableEl.querySelectorAll('.o_modified_image_to_save')].map(async el => {
+            const b64Proms = [...editableEl.querySelectorAll('.o_b64_image_to_save')].map(async el => {
+                // If image IS the original, it still needs to be saved. (e.g. dropped from outside)
```

Comment:
Line too long, maybe not useful anymore, at least I'd adapt the method docstring which would be more useful, as it's now no more true: `Create/Update cropped attachments.`